README.rst

```diff
@@ -9,6 +9,13 @@ Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
 Currently, Windows PowerShell, Command Prompt, and Git Shell for
 Windows are supported with limitations (See `Windows Issues`_).
 
+This product is supported by the Cybersecurity Development team at the 
+University of Illinois, on a best-effort basis. As of the last update to 
+this README, the expected End-of-Life and End-of-Support dates of this 
+version are October of 2025, the same as its primary dependencies: the 
```

Comment:
Do we want a specific date, or is the month and year enough?

Reply:
I think Month/Year is enough.